app/views/lookup_keys/_fields.html.erb

```diff
@@ -1,60 +1,61 @@
+    <h6><%= _("Override the Puppet class default value") %></h6>
+    <%= checkbox_f(f, :override, :onchange => 'toggleOverrideValue(this)', :size => "col-md-8",
+                   :help_inline => popover('', _('Whether the smart variable value is managed by Foreman'))
         ) if is_param %>
-    <%= validator_type_selector f  %>
-    <%= text_f f, :validator_rule, :size => "col-md-8", :disabled => f.object.validator_type.blank? %>
-    <div <%= "id=#{(f.object.key || 'new_lookup_keys').to_s.gsub(' ','_')}_lookup_key_override_merge" %> style=<%= "display:none;" if (!f.object.merge_overrides) %>>
-        <legend><%= _("Override merging options") %></legend>
-        <%= checkbox_f(f, :merge_overrides, :onchange => 'mergeOverridesChanged(this)',
-                          :disabled => !f.object.supports_merge?, :size => "col-md-8",
-                          :help_block => _("Should the matchers continue to look for matches after first find (only array/hash type). Note: merging overrides ignores all matchers that use Puppet default")) %>
-        <%= checkbox_f(f, :avoid_duplicates, :disabled => (!f.object.supports_uniq? || !f.object.merge_overrides), :size => "col-md-8",
-                          :help_block => _("Should the matched result avoid duplicate values (only array type).")) %>
-    </div>
-
-    <legend><%= _("Override value for specific hosts") %></legend>
-    <%= textarea_f f, :path, :rows => :auto, :label => _("Order"), :size => "col-md-8", :value => f.object.path,
-                      :help_block => popover(_("The order in which values are resolved"),
-                                             _("The order in which matchers keys are processed, first match wins.<br> You may use multiple attributes as a matcher key, for example, an order of <code>host group, environment</code> would expect a matcher such as <code>hostgroup = \"web servers\", environment = production</code>"), :title => _("The order in which values are resolved")).html_safe
-    %>
 
-    <div class="children_fields lookup_values">
-      <%= new_child_fields_template(f, :lookup_values, {:partial => "lookup_keys/value", :form_builder_attrs => {:is_param => is_param}}) %>
-      <%= f.fields_for :lookup_values do |lookup_values| %>
-        <%= render 'lookup_keys/value', :f => lookup_values, :is_param => is_param %>
-      <% end %>
-      <%= add_child_link '+ ' + _("Add Matcher-Value"), :lookup_values, { :title => _('add a new matcher-value pair')} %>
-      <%= documentation_button('4.2.6SmartMatchers') %>
-    </div>
+    <%= param_type_selector(f, :onchange => 'keyTypeChange(this)') %>
+    <%= textarea_f f, :default_value, :value => f.object.default_value_before_type_cast, :size => "col-md-8",
+                   :disabled => (f.object.is_param && (!f.object.override || f.object.use_puppet_default)),
+                   :fullscreen => :true,
+                   :rows => :auto, :help_inline => popover('', _("Value to use when there is no match")) %>
```

Comment:
do you still see them glued together?
after changing the css I see this:
![image](https://cloud.githubusercontent.com/assets/7894647/8569269/504513b8-257e-11e5-86b2-e2bb836660c1.png)
but maybe I added a margin that was too small?
https://github.com/theforeman/foreman/pull/2398/files#diff-4b492e48ae68ab49ffd1f96675dfa885R26